### ISSUE TITLE
Is the meta-genivi-dev-qtas deprecated in favor of PELUX SDE?

### ISSUE THREAD

**Issue by jeremiah** [Pelagicore/meta-genivi-dev-qtas#2]
If so, this git repo ought to point to PELUX and archived.

**Comment by tobsan** [comment]
Afaik, this layer and the PELUX SDE doesn't do the same things. From what I understand, this layer is meant to add QtAS onto GDP or something like that? Perhaps @erikboto or @rpannek can explain it better?

**Comment by rpannek** [comment]
Yes that is true @tobsan but there was never a actuall decision on what to do with the GDP flavaur of it, we updated it two times during the last 3 years just to get it working for a demo, otherwise nobody used it as far as I know.
In my personal opinion meta-genivi-dev-qtas should be deprecated, but I don't feel empowered enough to decide it because I might not have all the cases covered in my consideration.
